clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/view/AzureVMImageLookupController.groovy

```diff
@@ -61,15 +62,31 @@ class AzureVMImageLookupController {
   List<AzureNamedImage> getVMImage(@PathVariable String account, @PathVariable String region, @PathVariable String imageId) {
```

Comment:
Any particular reason we shouldn't extend Rosco to support baking Azure images? I recall we discussed this briefly a while back, and there does exist an Azure builder for Packer.